### DISCUSSION TITLE
怎么只编译6.1内核的固件？

### DISCUSSION THREAD

**Discussion by rmoyulong** [immortalwrt/immortalwrt#1243] [Q&A]
### Describe the bug

我的CONIFIG文件里面定义了6_1=y，为何还是编译出来6.6的固件？
请高手解答

### ImmortalWrt version

CONFIG_TARGET_qualcommax_ipq807x=y

### ImmortalWrt release

master

### ImmortalWrt target/subtarget

master

### Device

CONFIG_TARGET_qualcommax_ipq807x=y

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by xlighting2017** [comment]
因为6.1已经过时了，现在只有6.6

**Reply by xlighting2017** [reply]
自己编
不会可以学
不学可以等

你没有 不等于 不存在
